PMSCore/Tester.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace PMSCore
+
+{
+    /// <summary>
+    /// A Tester user who managages test cases and bug reporting.
+    /// </summary>
+    public class Tester : UserBase
+    {
+        
+
+        // Properties specific to a Tester
+        public List<string> TestCases;
+        public List<(string Description, string Severity)> BugReports;
+
+        /// <summary>
+        /// Initializes a Tester and assigns username & TESTER permission level
+        /// </summary>
+        public Tester(string userName) : base(userName, Permission.TESTER)
+        {
+            TestCases = new List<string>();
+            BugReports = new List<(string, string)>();
+        }
+
+
+        public void SetPermission(Permission permission)
+        {
+            UserPermission = permission; 
+        }
+
+        public Permission GetPermission()
+        {
+            return UserPermission;
+        }
+
+        // Tester-Specific Methods
+
+        /// <summary>
+        /// Adds a new test case to the tester's list of test cases.
+        /// </summary>
+        public void AddTestCase(string testCaseDescription)
+        {
+            if (!string.IsNullOrWhiteSpace(testCaseDescription))
+            {
+                TestCases.Add(testCaseDescription);
+                Console.WriteLine($"Test case added: {testCaseDescription}");
+            }
+            else
+            {
+                Console.WriteLine("Test case description should not be empty.");
+            }
+        }
+
+        /// <summary>
+        /// Method to view all test cases created by the tester.
+        /// </summary>
+        public void ViewTestCases()
+        {
+            Console.WriteLine("Test Cases:");
+            foreach (var testCase in TestCases)
+            {
+                Console.WriteLine($"- {testCase}");
+            }
+        }
```

Comment:
these lines:             
            
            Console.WriteLine("Test Cases:");
            foreach (var testCase in TestCases)
            {
                Console.WriteLine($"- {testCase}");
            }
        }

ViewTestCases() can be changed to return a list of test cases or a formatted string because this way it directly writes output to the console, and that's not good for testing.